### PR TITLE
update license

### PR DESCRIPTION
**Pull request by pragmatos**

### CODE REVIEW

**Review by OBrezhniev — APPROVED** [review]

File: LICENSE-APACHE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2023 0kims Association
```
Comment (OBrezhniev) [inline LICENSE-APACHE:189]: All 0xPolygonID repos should have (c) ZKID Labs AG, not 0kims Association

File: LICENSE-MIT [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright 2023 0kims Association.
```
Comment (OBrezhniev) [inline LICENSE-MIT:3]: Same here

**Review by OBrezhniev — CHANGES_REQUESTED** [review]

File: LICENSE-MIT (hunk 2 above)
Comment (pragmatos) [reply LICENSE-MIT]: fixed

**Review by vmidyllic — DISMISSED** [review]

File: LICENSE-APACHE [hunk 3]
```diff
@@ -0,0 +1,202 @@
+Copyright (C) 2024  ZKID Labs AG
```
Comment (OBrezhniev) [inline LICENSE-APACHE:1]: You have copyright at the bottom, why do you need another one at the top?

Comment (OBrezhniev) [reply LICENSE-APACHE:1]: @pragmatos please remove this line.

**Review by OBrezhniev — APPROVED** [review]

**Review by vmidyllic — APPROVED** [review]

**Pull request merged**